### PR TITLE
fix(curriculum): add note to clarify instructions

### LINKED ISSUE #54003: Explain in this step that the HTML entity won't work with innerText
### Describe the Issue

Edit: Updated issue.

The camper is asked to add the `"You die. &#x2620;"` to the `text` property which won't work with this step's current code.

As pointed out this will be fixed in a later step but I question leaving it as is. It is just going to be confusing. I would suggest updating this step with a comment about it so it isn't misinterpreted as a bug.

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-basic-javascript-by-building-a-role-playing-game/step-136

### Your code

```html
<p class="nope"></p>
<p class="yes"></p>
```

```js
document.querySelector('.nope').innerText = "&#x2620;"
document.querySelector('.yes').innerHTML = "&#x2620;"
```

### Expected behavior

Acknowledge the flaw in the code in this step and mention it will be fixed later.

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

Forum: https://forum.freecodecamp.org/t/learn-basic-javascript-by-building-a-role-playing-game-step-136-large-bug/677592

### PR DESCRIPTION
**Pull request by Supravisor**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #54003

### CODE REVIEW

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62ba17beef16c563069a65d8.md [hunk 1]
```diff
@@ -9,6 +9,8 @@ dashedName: step-136
 
 In the `locations` array, add another object at the end. Set the `name` property to `"lose"`, set `"button text"` to an array with three `"REPLAY?"` strings, set `"button functions"` to an array with three `restart` variables, and set `text` to `"You die. &#x2620;"`. 
 
+In a later step, you will update a function for the &#x2620; emoticon text to properly display on the page.
```
Comment (lasjorg) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62ba17beef16c563069a65d8.md:12]:
```suggestion
In a later step, you will fix the code in the `update` function to make the `&#x2620;` emoticon text properly display on the page.
```

How about this?

I think we at least need to add a code block for the emoticon text value.

**Review by lasjorg — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62ba17beef16c563069a65d8.md (hunk 1 above)
Comment (Supravisor) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62ba17beef16c563069a65d8.md:12]: Happy to add back ticks for the emoticon code.
Will leave wording of new text for fCC staff to confirm.

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62ba17beef16c563069a65d8.md [hunk 2]
```diff
@@ -9,6 +9,8 @@ dashedName: step-136
 
 In the `locations` array, add another object at the end. Set the `name` property to `"lose"`, set `"button text"` to an array with three `"REPLAY?"` strings, set `"button functions"` to an array with three `restart` variables, and set `text` to `"You die. &#x2620;"`. 
 
+In a later step, you will update a function for the `&#x2620;` emoticon text to properly display on the page.
```
Comment (lasjorg) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62ba17beef16c563069a65d8.md:12]: I'm just not crazy about the "update a function" part. How about just "the code" instead?
```suggestion
In a later step, you will update the code for the `&#x2620;` emoticon text to properly display on the page.
```

**Review by lasjorg — CHANGES_REQUESTED** [review]

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62ba17beef16c563069a65d8.md (hunk 2 above)
Comment (Sembauke) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62ba17beef16c563069a65d8.md:12]: I would like this change to be added.

Comment (jdwilkin4) [reply curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62ba17beef16c563069a65d8.md]: I committed those new changes

**Review by gikf — APPROVED** [review]

**Review by jdwilkin4 — APPROVED** [review]

**Review by lasjorg — APPROVED** [review]

**Pull request merged**